runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -381,13 +438,13 @@ def StartDebug_term(dict: dict<any>)
   var counter_max = 300
   var success = false
   while success == false && counter < counter_max
-    if CheckGdbRunning() != 'ok'
-      # Failure. If NOK just return.
+    if IsGdbStarted() == false
```

Comment:
When using a Boolean variable, there is no need to it compare against true or false. Can you use `!IsGdbStarted()`?

Reply:
Done! I extended the process also in non-cited places that suffer of the same issue.